docs/pipelines/scripts/git-commands.md

```diff
@@ -76,10 +76,10 @@ On the **Version Control** tab, select the repository in which you want to run G
 
 Grant permissions needed for the Git commands you want to run. Typically you'll want to grant:
 
-* **Branch creation:**  Allow
+* **Create branch:**  Allow
 * **Contribute:**  Allow
-* **Read:**  Inherited allow
-* **Tag creation:**  Inherited allow
+* **Read:**  Allow (system)
+* **Create tag:**  Allow (system)
```

Comment:
@amang2205 , thank you for the contribution! The only thing I see is that I believe the `Create Tag` permission is just `Allow` I don't see `Allow (system)`. Is this correct for you as well?

Reply:
You're correct, an update of the UI has resolved in that change.